src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -229,6 +242,9 @@ export default class Aircraft {
         this.heading = _get(data, 'heading', this.heading);
         this.altitude = _get(data, 'altitude', this.altitude);
         this.speed = _get(data, 'speed', this.speed);
+
+        this._f = new Fms(data, this.initialRunwayAssignment, this._navigationLibrary);
```

Comment:
very much so.  this is the _'I want to get it working without unhooking the previous implementation that has the name I was to use"_ trick